### ISSUE TITLE
Screen Reader: Track Distance: The button 'refer your boss' grouped with the button 'x'

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76902]
## Action Performed:
Prerequisites:
- The user is signed in
- The start and stop points have been selected on the track distance page

1. Using Chrome + JAWS, open the URL - https://new.expensify.com/
2. Navigate to the "+" icon and press Enter
3. Tab to the "Track distance" menu item and press Enter to open the Pop-up
4. After selecting the "Start/stop" points, Tab to the "Next" button and press Enter
5. Tab to the "Expensify.org / Teachers Unite" and press Enter to open
6. Tab to the "refer your boss" and the close "X" button
7. Observe the screen reader announcement

## Expected Result:
Each interactive element must expose a correct accessible role, name, and be announced individually.

Here, each button should be announced separately:
- 'Submit an expense, refer your boss button'
- 'Close button'

## Actual Result:
The 'refer your boss' button is announced without a role as 'Submit an expense, refer your boss', so that the screen reader users cannot identify it as a button element.
The close 'X' button is grouped with the control together, and it is announced as 'referral close button'.

HTML:
<div aria-label="referral" role="presentation" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-1xfd6ze r-hcv6sl r-18u37iz r-1wtj0ep r-1awozwy r-b5h31w r-1q142lx" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx" style="gap: 10px; padding: 10px;"><div class="css-g5y9jx r-1habvwh r-kc3elw r-1xnzce8 r-dnmrzs" data-testid="html" style="flex: 1 1 0%;"><div class="css-g5y9jx r-1habvwh r-kc3elw r-1xnzce8 r-dnmrzs" data-testid="body"><div dir="auto" class="css-146c3p1 r-1xnzce8" style="font-size: 15px; color: rgb(231, 236, 233); font-family: &quot;Expensify Neue&quot;; font-style: normal; font-weight: 400; line-height: 20px; direction: ltr;"><span class="css-1jxf684 r-1xnzce8" style="font-size: 15px; color: rgb(231, 236, 233); font-family: &quot;Expensify Neue&quot;; font-style: normal; font-weight: 400; line-height: 20px; direction: ltr;">Submit an expense, </span><span class="css-1jxf684 r-1xnzce8" data-testid="success" style="font-size: 15px; color: rgb(3, 212, 124); font-family: &quot;Expensify Neue&quot;; font-style: normal; font-weight: 400; line-height: 20px; direction: ltr;"><span class="css-1jxf684 r-1xnzce8" data-testid="strong" style="font-size: 15px; color: rgb(3, 212, 124); font-family: &quot;Expensify Neue&quot;; font-style: normal; font-weight: 700; line-height: 20px; direction: ltr;">refer your boss</span></span><span class="css-1jxf684 r-1xnzce8" style="font-size: 15px; color: rgb(231, 236, 233); font-family: &quot;Expensify Neue&quot;; font-style: normal; font-weight: 400; line-height: 20px; direction: ltr;">.</span></div></div></div><button aria-label="Close" role="button" data-selection-scrapper-hidden-element="true" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-1awozwy r-eu3ka r-1777fci r-1aockid r-9ffhgg r-lrvibr" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#8B9C8F"><path d="M14.6 17.4c.8.8 2 .8 2.8 0s.8-2 0-2.8L12.8 10l4.6-4.6c.8-.8.8-2 0-2.8s-2-.8-2.8 0L10 7.2 5.4 2.6c-.8-.8-2-.8-2.8 0s-.8 2 0 2.8L7.2 10l-4.6 4.6c-.8.8-.8 2 0 2.8s2 .8 2.8 0l4.6-4.6z"></path></svg></div></button></div>

## Area issue was found in:
Track Distance

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Ensure the 'refer your boss' button and the close 'X' button are coded as two separate interactive elements, not grouped or nested inside the same element
Remove the aria-label and tabindex="0" from the wrapping <div>. Ensure the button 'refer your boss' is implemented as a separate button, so it focuses and announces separately from the button 'x'.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes

## Environment(s)
Windows Windows 11 Home - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/f510c510-639c-462a-9901-78b771746020

<img width="1919" height="966" alt="Image" src="https://github.com/user-attachments/assets/a9e73b3f-1efd-49df-8992-b70d77c02c42" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6996996
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by ahmedGaber93** [comment]
Coming from #86510, the new structure for `ReferralProgramCTA` made the “Refer a friend” tap target feel unresponsive.

We fixed this by adding pointerEvents="none" to the View wrapping RenderHTML, ensuring all touch events pass through the text layer to the underlying absolutely positioned Pressable.
